demisto_sdk/commands/lint/linter.py

```diff
@@ -816,7 +815,8 @@ def _docker_run_pytest(self, test_image: str, keep_container: bool, test_xml: st
                     with open(file=xml_apth, mode='bw') as f:
                         f.write(test_data_xml)  # type: ignore
 
-                if cov:
+                if not no_coverage:
+                    cov_file_path = os.path.join(self._pack_abs_dir, '.coverage')
```

Comment:
1. please use Pathlib
2. please make sure the file exists and handle when it doesn't